### PR TITLE
Server/delete card

### PR DESCRIPTION
**Pull request by nikitasakau**
Tried do through
```
Card
   .destroy({
     where: { id: req.params.id },
   })
   .then(() => { res.status(204).send() })
   .catch((error) => { res.status(400).send(error) })
```
But destroy method always send `204`, even if this card not exist. So, thats why used findByPk.

### CODE REVIEW

**Comment by nikitasakov** [comment]
[I as user want to delete TODO card](https://trello.com/c/0JrKBanN/12-i-as-user-want-to-delete-todo-card)

File: server/src/controllers/v1/cards.js [hunk 1]
```diff
@@ -10,7 +10,13 @@ module.exports = {
...
   },
-
+  destroy: (req, res) => {
+    Card
+      .findByPk(req.params.id)
+      .then((card) => { card.destroy() })
+      .then(() => { res.status(204).send() })
+      .catch(() => { res.status(400).send({ error: 'Bad Request', message: 'Card not found' }) })
+  },
   getAll: (req, res) => {
```
Comment (AleksSenkou) [inline server/src/controllers/v1/cards.js]: @nikitasakov can we leave empty line between actions? So the code can be more readable

File: server/src/controllers/v1/cards.js [hunk 2]
```diff
@@ -10,7 +10,13 @@ module.exports = {
       .then((card) => { res.status(201).send(card) })
       .catch((error) => { res.status(400).send(error) })
   },
-
+  destroy: (req, res) => {
+    Card
+      .findByPk(req.params.id)
+      .then((card) => { card.destroy() })
+      .then(() => { res.status(204).send() })
+      .catch(() => { res.status(400).send({ error: 'Bad Request', message: 'Card not found' }) })
```
Comment (AleksSenkou) [inline server/src/controllers/v1/cards.js]: @nikitasakov can't we get this error `'Card not found'` from Sequalize API?

File: server/src/controllers/v1/cards.js [hunk 3]
```diff
@@ -10,7 +10,13 @@ module.exports = {
       .then((card) => { res.status(201).send(card) })
       .catch((error) => { res.status(400).send(error) })
   },
-
+  destroy: (req, res) => {
+    Card
+      .findByPk(req.params.id)
+      .then((card) => { card.destroy() })
```
Comment (AleksSenkou) [inline server/src/controllers/v1/cards.js]: `.then((card) => { card.destroy() })`

can be used without `{}` and `()` parentheses

`.then(card => card.destroy())`

Comment (AleksSenkou) [reply server/src/controllers/v1/cards.js]: @datarockets/cuties thoughts about preventing `{}` and `()` parentheses usage where possible?

**Review by AleksSenkou — COMMENTED** [review]
Looks good, here are some comments:

File: server/src/controllers/v1/cards.js (hunk 3 above)
Comment (AleksSenkou) [reply server/src/controllers/v1/cards.js]: @DDKatch I don't have sublime linter for that too 😄

Comment (DDKatch) [reply server/src/controllers/v1/cards.js]: hah, in react part it'll be fixed automatically, but for backend part it doesn't work so 🤔 @AleksSenkou what do you think about PP for linters for everyone, in order to fix this ?

Comment (AleksSenkou) [reply server/src/controllers/v1/cards.js]: @DDKatch what do you mean by `for backend part it doesn't work so`? Kind of

`.then(card => card.destroy())` will throw an error?

Comment (DDKatch) [reply server/src/controllers/v1/cards.js]: > .then((card) => { card.destroy() })
> can be used without {} and () parentheses
> .then(card => card.destroy())

@AleksSenkou I mean my linter automatically fix this case for react part

Comment (AleksSenkou) [reply server/src/controllers/v1/cards.js]: @DDKatch got it 👍 

PP works well for me. Any way we can firstly fo with infra task. @DDKatch can you please add one?

Comment (DDKatch) [reply server/src/controllers/v1/cards.js]: @AleksSenkou yes, for sure

**Review by DDKatch — APPROVED** [review]
LGTM

**Comment by AleksSenkou** [comment]
LGTM

**Pull request merged**